langspec/xproc30/xproc.xml

```diff
@@ -4932,10 +4932,24 @@ as defined below, are removed:</para>
               prefix has been preserved, but it would be equally correct if a different
               prefix had been used.</para>
 
-<para>If the <tag class="attribute">expand-text</tag> attribute is
-not false, then each text node within the <tag>p:inline</tag> is evaluated
-as a <glossterm>text value template</glossterm>. The context node for
-these expressions is undefined.</para>
+<para>If the <tag class="attribute">expand-text</tag> attribute is not
+false, then each attribute value and text node within the
+<tag>p:inline</tag> is evaluated as a <glossterm>text value
+template</glossterm>. The context node for these expressions is
+undefined.</para>
+
+<para>Within the descendants of <tag>p:inline</tag> the attribute
+<tag class="attribute">p:expand-text</tag> (i.e., the attribute <code>expand-text</code>
+<emphasis>in the XProc namespace</emphasis>), may appear on an element.
+If it does, the specified value is used as the expand text value
+for the descendants of the element on which it occurs. The
+<tag class="attribute"> will not appear in the inline content.</tag>
```

Comment:
→ `<tag class="attribute">p:expand-text</tag>` will not appear in the inline content.

Reply:
Fixed.